### PR TITLE
New resource vpa for deployment controller resources

### PR DESCRIPTION
**Pull request by santinoncs**
#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [ ] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

File: stable/nginx-ingress/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: nginx-ingress
-version: 1.19.1
+version: 1.20.1
```
Comment (taharah) [inline stable/nginx-ingress/Chart.yaml]: This should be bumped to `1.21.0` now since another PR was merged earlier today.

**Review by taharah — CHANGES_REQUESTED** [review]
Isn't the `VerticalPodAutoscaler` CRD definition missing? As it stands, the values for the VPA definition are just being defined, but not utilized anywhere.

File: stable/nginx-ingress/Chart.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: nginx-ingress
-version: 1.24.3
+version: 1.24.4
```
Comment (ChiefAlexander) [inline stable/nginx-ingress/Chart.yaml]: as this is a new feature this version should be `1.25.0`

**Pull request closed without merging**